charts/argo-workflows/templates/controller/workflow-controller-crb.yaml

```diff
@@ -41,5 +41,12 @@ subjects:
   - kind: ServiceAccount
     name: {{ template "argo-workflows.controllerServiceAccountName" . }}
     namespace: {{ include "argo-workflows.namespace" . | quote }}
+{{- if .Values.controller.clusterWorkflowTemplates.serviceAccounts }}
+{{- range .Values.controller.clusterWorkflowTemplates.serviceAccounts }}
```

Comment:
nits: I thinks we don't need `if` condition ..? 👀 
```diff
-{{- if .Values.controller.clusterWorkflowTemplates.serviceAccounts }}
{{- range .Values.controller.clusterWorkflowTemplates.serviceAccounts }}
  - kind: ServiceAccount
    name: {{ .name }}
    namespace: {{ .namespace | quote }}
{{- end }}
-{{- end }}
```

Reply:
nice catch

Reply:
Thanks, I didn't notice it!
I removed the if statement from both the CRB and RB templates :D

Reply:
Unfortunatly it seems you've angered the DCO checker now :(

Reply:
Yeah I forgot to sign my commit, I ran `--amend`so now it's good 😎